.vscode/tasks.json

```diff
@@ -1,6 +1,164 @@
 {
   "version": "2.0.0",
   "tasks": [
+    {
+      "label": "install (CopilotChatApp)",
+      "type": "shell",
+      "group": "build",
+      "command": "yarn",
+      "presentation": {
+        "echo": true,
+        "reveal": "always",
+        "showReuseMessage": false,
+        "panel": "shared",
+        "group": "buildTasks"
+      },
+      "options": {
+        "cwd": "${workspaceFolder}/samples/apps/copilot-chat-app/webapp"
+      },
+      "problemMatcher": []
+    },
+    {
+      // build using yarn
+      "label": "build (CopilotChatApp)",
+      "type": "shell",
+      "group": "build",
+      "command": "yarn build",
+      "presentation": {
+        "echo": true,
+        "reveal": "always",
+        "panel": "shared",
+        "group": "buildTasks"
+      },
+      "options": {
+        "cwd": "${workspaceFolder}/samples/apps/copilot-chat-app/webapp"
+      },
+      "problemMatcher": []
+    },
+    {
+      // yarn start
+      "label": "start (CopilotChatApp)",
+      "type": "shell",
+      "group": "test",
+      "command": "yarn start",
+      "presentation": {
+        "reveal": "always",
+        "panel": "shared",
+        "group": "copilot"
+      },
+      "options": {
+        "cwd": "${workspaceFolder}/samples/apps/copilot-chat-app/webapp"
+      }
+    },
+    //semantic-kernel/samples/apps/copilot-chat-app/webapi
+    // build
+    {
+      "label": "build (CopilotChatApi)",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/samples/apps/copilot-chat-app/webapi/CopilotChatApi.csproj",
+        "/property:GenerateFullPaths=true",
+        "/consoleloggerparameters:NoSummary",
+        "/property:DebugType=portable"
+      ],
+      "problemMatcher": "$msCompile",
+      "group": "build"
+    },
+    // run
+    {
+      "label": "run (CopilotChatApi)",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "run",
+        "--project",
+        "${workspaceFolder}/samples/apps/copilot-chat-app/webapi/CopilotChatApi.csproj"
```

Comment:
```
# Overwrite existing .env file
Set-Content -Path $EnvFilePath -Value "REACT_APP_BACKEND_URI=https://localhost:40443/"

if ($TenantId -eq 'msa') 
{
    $TenantId = '9188040d-6c67-4c5b-b112-36a304b66dad'
}

Add-Content -Path $EnvFilePath -Value "REACT_APP_AAD_AUTHORITY=https://login.microsoftonline.com/$TenantId"
Add-Content -Path $EnvFilePath -Value "REACT_APP_AAD_CLIENT_ID=$ClientId"
```

Add additional input functionality like in #810

Reply:
I'd also vote to do this as a follow up improvement. Not able to focus on it right now.

Reply:
Added todo.